crates/bitwarden-exporters/src/client_exporter.rs

```diff
@@ -6,11 +6,11 @@ use crate::{
     ExportError, ExportFormat,
 };
 
-pub struct ClientExporters<'a> {
+pub struct ExporterClients<'a> {
```

Comment:
The plural name doesn't fit any longer since it's a single client, but multiple exporters. I think we can use `ExporterClient`.

Reply:
Addressed in 631ec158e834891a1c16438bdf92fcb13d100efd.